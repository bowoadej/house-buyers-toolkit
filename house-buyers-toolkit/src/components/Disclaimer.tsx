export default function Disclaimer() {
    return (
        <p>NOTE:This information is purely for informative purposes only. It does not consitute as financial information and does not consistiute a contract. Please contact a certified financial professional for financial advice.</p>
    )
}