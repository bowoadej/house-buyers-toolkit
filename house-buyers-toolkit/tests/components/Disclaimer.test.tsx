import renderer from 'react-test-renderer';
import { expect, it } from '@jest/globals'
import Disclaimer from '../../src/components/Disclaimer'

it('renders correctly', () => {
    const tree = renderer
        .create(<Disclaimer />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});