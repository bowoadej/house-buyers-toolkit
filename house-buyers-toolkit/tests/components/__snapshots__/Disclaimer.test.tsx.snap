// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p>
  NOTE:This information is purely for informative purposes only. It does not consitute as financial information and does not consistiute a contract. 
  <br />
   Please contact a certified financial professional for financial advice.
</p>
`;
